modules/dhcp_common/free_ips.rb

```diff
@@ -0,0 +1,158 @@
+require 'concurrent'
+require 'logger'
+require 'set'
+
+module Proxy::DHCP
+  class FreeIps
+    include ::Proxy::Log
+
+    DEFAULT_ADDRESS_KEEP_TIME = 30*60 # 30 mins
```

Comment:
I only have one tiny nitpick - this constant does only seem to be used in tests but in the default value you don't use it. Is this intentional? Might be some leftover.

Reply:
Removed.